components/searching-consultant/filters/FilterModalMain.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Text } from 'react-native';
+import SpecialtyModalContent from './filters-tabs/SpecialtyModal';
+import ExperienceModalContent from './filters-tabs/ExperienceModal';
+import RegionModalContent from './filters-tabs/RegionModal';
+import PriceRangeModalContent from './filters-tabs/PriceRangeModal';
+import AllFiltersModal from '../master-filter/masterFilter';
+import tabs from '../../../locales/en/Tabs.json';
+//Depending on the filterCategory This component is used to render a specific modal.
+const FilterModalContent = ({ filterCategory, closeModal }) => {
```

Comment:
Nitpick: the component's name and the file's name should be the same, especially when they are so many Filters and Modals component around here